modules/argocd-machinesets-sample.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-machinesets-sample_{context}"]
+= MachineSets
+
+You can use ArgoCD to deploy custom MachineSets.
+
+.Prerequisites
+
+* Obtain a MachineSet for your cluster.
+
+.Procedure
+
+
+. Create the `machineset` application in ArgoCD:
++
+----
+$ argocd app create machineset --repo https://github.com/<account_name>/openshift4-gitops.git --path=machine-sets --dest-server=https://kubernetes.default.svc --dest-namespace=openshift-machine-api
```

Comment:
$argocd app create machineset --repo https://github.com/<account_name>/openshift4-gitops.git --path=machine-sets --dest-server=https://kubernetes.default.svc --dest-namespace=openshift-machine-api
About this command, the value  "openshift-machine-api" of "--dest-namespace" parameter needs to be modified to "openshift-config". If the value is openshift-machine-api, The machineSet can't be deployed successfully. I have create a bug in Jira https://issues.redhat.com/browse/CO-763